src/NHSD.BuyingCatalogue.Ordering.Api/Controllers/OrderingPartyController.cs

```diff
@@ -0,0 +1,63 @@
+﻿using System;
+using System.ComponentModel.DataAnnotations;
+using Microsoft.AspNetCore.Mvc;
+using NHSD.BuyingCatalogue.Ordering.Api.Models;
+
+namespace NHSD.BuyingCatalogue.Ordering.Api.Controllers
+{
+    [Route("api/v1/orders/{orderId}/ordering-party")]

```

Comment:
Sorry Ruth this should have been `/sections/ordering-party` if it wasn't on the JSON contract.

Reply:
I dont think it was, but no worries 👍